anvil_consortium_manager/models.py

```diff
@@ -79,6 +82,30 @@ def anvil_import(cls, billing_project_name):
         billing_project.save()
         return billing_project
 
+    @classmethod
+    def anvil_audit(cls):
```

Comment:
My thinking is that there can be class methods that check records for that class against what's on AnVIL. Then I can add a management command that calls all of those class methods and possibly produces a report.

Reply:
I'm now wondering if this method should move to the `AnVILAuditResult` subclasses (and maybe rename them to just `AnVILAudit`). I tried adding a `model_type` property to `AnVILAuditResults` so that `AccountAuditResults` could check that any model instance passed to it was an `Account`. That means that `anvil_audit.py` needs to import `models.py`, which ended up being a circular import because `models.py` imports `anvil_audit.py` (so that I could write the `anvil_audit` class methods). If I move the `anvil_audit` methods to the `AnVILAuditResults` subclasses, I could both specify the model type and avoid a circular import without too many changes to the code.